src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -409,6 +409,151 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
     return oldMinimumStaffEntriesWidth * elongationFactorForMeasureWidth;
   }
 
+  public calculateMeasureWidthFromChords(
+    measuresVertical: GraphicalMeasure[],
+    oldMinimumStaffEntriesWidth: number
+  ): number {
+    let elongationFactorForMeasureWidth: number = 1;
+
+    interface ChordEntryInfo {
```

Comment:
Was the definition of interfaces inside the class on purpose? I'd assume they should be either parallel or in a separate file.

Reply:
Good question.  I'm not sure really...  I used the function that does the lyrics as a template. I imagine they put the interfaces there to keep it simple?